scrypto/src/engine/wasm_api.rs

```diff
@@ -61,8 +61,13 @@ extern "C" {
         args_len: usize,
     ) -> Buffer;
 
-    /// Invokes any function or method, for both Scrypto and Native blueprints.
-    pub fn invoke(invocation_ptr: *const u8, invocation_len: usize) -> Buffer;
+    /// Invokes a native function or method.
+    pub fn call_native(
```

Comment:
What is call_native again in this case? Is it still the idea that we are moving away from the client knowing the difference between native vs. non-native? ie, this is a preparatory split before merge?

Reply:
Yes, it's preparatory. I'm incrementally downscaling the `call_native` until the backend is fully prepared.

```
invoke_scrypto (removed in this PR) + invoke_native_blueprint + invoke_node_module
```
Our step is probably to remove `invoke_native_blueprint`, which seems to be the easiest.